src/ches/emailHelpers.ts

```diff
@@ -24,6 +24,13 @@ export const quotaChanged = (project, requestedProject) => {
     );
 };
 
+export const budgetChanged = (project, requestedProject) => {
+    return !(
```

Comment:
I understand what the function does, but as Javascript's object keys are not always ordered, the stringified JSON strings might be different. We could consider using `lodash _isEqual`:
- https://lodash.com/docs/4.17.15#isEqual
- also we can avoid including whole `lodash` library by importing the function only:
   https://www.npmjs.com/package/lodash.isequal